src/Microsoft.Win32.Primitives/src/Properties/AssemblyInfo.cs

```diff
@@ -26,5 +26,4 @@
 [assembly: NeutralResourcesLanguageAttribute("en-US")]
 
 [assembly: CLSCompliant(true)]
-[assembly: ComVisible(false)]
```

Comment:
Thanks, Wes. Yes, I'll hold off on merging until the mirror is enabled. Sounds like we shouldn't need to add anything, but good to know we'll be able to do it in a single place if necessary.